### PR TITLE
Features/173 introduce bizual 2nd model monitor inventory direct from warehouse

### PR DESCRIPTION
**Pull request by RoiJS**
Updates related with 2nd model

### CODE REVIEW

File: Beelina.APP/src/app/_guards/access.guard.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import { Observable } from 'rxjs';
+import { SidedrawerService } from '../_services/sidedrawer.service';
+import { AuthService } from '../_services/auth.service';
+import { NotificationService } from '../shared/ui/notification/notification.service';
+
+@Injectable()
+export class AccessGuard {
+  constructor(
+    private authService: AuthService,
```
Comment (roam915) [inline Beelina.APP/src/app/_guards/access.guard.ts:17]: Unused angular service. Kindly remove.

**Review by roam915 — CHANGES_REQUESTED** [review]
Done checking commits. Check one comment.

File: Beelina.APP/src/app/_guards/access.guard.ts (hunk 1 above)
Comment (RoiJS) [reply Beelina.APP/src/app/_guards/access.guard.ts:17]: Well, its actually used when automatically logging out the user if unauthorized to access the page. **NO ACTION**

Comment (roam915) [reply Beelina.APP/src/app/_guards/access.guard.ts:17]: Yeah you are right. Then pull request is now approved!

**Review by roam915 — APPROVED** [review]
Approved now!

**Pull request merged**